VAMobile/src/screens/HealthScreen/TravelPay/TravelPayClaims/TravelPayClaimsFilter.tsx

```diff
@@ -0,0 +1,180 @@
+}: TravelPayClaimsFilterProps) {
+  const { t } = useTranslation(NAMESPACE.COMMON)
+  const theme = useTheme()
+
+  // TODO - need to replace with actual data from props or state
+  const currentFilter = 'All'
+  const currentSort = 'most recent'
+
+  const travelClaimsListTitle = () => {
+    const keys = {
+      count: totalClaims,
+      filter: currentFilter,
+      sort: currentSort,
+    }
+
+    return t('travelPay.statusList.list.title', keys)
+  }
+
+  const getPickerOptions = (): Array<TravelClaimsDatePickerOption> => {
+    const todaysDate = DateTime.local()
+    const threeMonthsEarlier = todaysDate.minus({ months: 3 })
+
+    return [
+      {
+        label: t('travelPay.statusList.pastThreeMonths'),
+        value: t('travelPay.statusList.pastThreeMonths'),
+        a11yLabel: t('travelPay.statusList.pastThreeMonths'),
+        dates: { startDate: threeMonthsEarlier.startOf('day'), endDate: todaysDate.endOf('day') },
+      },
+    ]
+  }
+
+  const pickerOptions = getPickerOptions()
+  const [datePickerOption, setDatePickerOption] = useState(pickerOptions[0])
+
+  const setValuesOnPickerSelect = (selectValue: string): void => {
+    const curSelectedRange = pickerOptions.find((el) => el.value === selectValue)
+    if (curSelectedRange) {
+      setDatePickerOption(curSelectedRange)
+    }
+  }
+
+  // TODO- add modal props
+  const modalProps: RadioGroupModalProps = {
+    groups: [
+      {
+        items: [
+          { optionLabelKey: 'All', value: 'all' },
+          { optionLabelKey: 'Paid', value: 'paid' },
+          { optionLabelKey: 'Denied', value: 'denied' },
+        ],
+        onSetOption: (newFilter: string) => {
+          setSelectedFilter(newFilter)
+        },
+        selectedValue: selectedFilter,
+        title: 'Filter by',
+      },
+      {
+        items: [
+          { optionLabelKey: 'Most recent', value: 'recent' },
+          { optionLabelKey: 'Oldest', value: 'oldest' },
+        ],
+        onSetOption: (newSortBy: string) => {
+          setSelectedSortBy(newSortBy)
+        },
+        selectedValue: selectedSortBy,
+        title: 'Sort by',
+      },
+    ],
+    buttonText: t('filterAndSort'),
+    buttonA11yLabel: t('filterAndSort'),
+    buttonTestID: 'openFilterAndSortTestID',
+    headerText: t('filterAndSort'),
+    onApply: () => {
+      // TODO- add apply logic
+      console.log('Apply filters:', selectedFilter, selectedSortBy)
+    },
+    onCancel: () => {
+      // TODO- add cancel logic
+      console.log('Cancel filters')
+    },
+    testID: 'TravelPayFilterModalTestID',
+  }
+
+  const filterContainerProps: BoxProps = {
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    mx: theme.dimensions.gutter,
+    mb: theme.dimensions.standardMarginBetween,
+  }
+
+  const onClearFiltersPress = () => {
+    // TODO- add clear filters functionality
+    console.log('Clear filters pressed')
```

Comment:
Could we use a mock function in a utils file so that we can test it easier like `mockClearFilters` then we can spy on it rather than a console.log?
```suggestion
    mockClearFilters()
```

Reply:
In that case I would probably not test this yet until we have the actual functionality to test

Reply:
OK - removed this test for now